src/ariel-os-embassy-common/src/ble.rs

```diff
@@ -1,5 +1,31 @@
 //! Common BLE types to be used across different HALs.
 
+use static_cell::StaticCell;
+use trouble_host::HostResources;
+
+/// Maximum Transmission Unit (MTU) for BLE connections. 27 should work for all BLE versions.
+///
+/// Since bluetooth 4.2, this can be increased to 251 bytes.
+// TODO: add the ability to configure this value.
+pub const MTU: usize = 27;
+
+/// Maximum number of concurrent connections to be handled by the BLE stack, minimum 1.
+pub const MAX_CONNS: usize = 1;
+/// Maximum number of concurrent channels to be handled by the BLE stack (not including GATT), minimum 1.
+pub const MAX_CHANNELS: usize = 1;
+
+static HOST_RESOURCES: StaticCell<BleHostResource> = StaticCell::new();
+
+/// Alias for the BLE host resources, setting the generic parameters to the constants for convenience.
+pub type BleHostResource = HostResources<MAX_CONNS, MAX_CHANNELS, MTU>;
```

Comment:
Shouldn't that be plural?

```suggestion
pub type BleHostResources = HostResources<MAX_CONNS, MAX_CHANNELS, MTU>;
```